c2corg_ui/templates/route/helpers/detailed_route_attributes.html

```diff
@@ -4,22 +4,24 @@
 ## LOCATION
 <%def name="get_route_location(route)">\
   % if route.get('areas') or route.get('orientations'):
-    <div class="name-icon-value location" ng-click="detailsCtrl.openTab($event)">
+  <div class="name-icon-value location
+       % if route.get('areas') and not route.get('orientations'):
+         ng-hide
+       % endif
+       "
```

Comment:
Could you use if else shortand : ${'ng-hide' if route.get('areas') and not route.get('orientations')}